dandi/bids_validator_xs.py

```diff
@@ -195,14 +217,12 @@ def load_top_level(
         # None value gets passed as list of strings...
         extensions = top_level_file["extensions"]
         if extensions != ["None"]:
-            periodsafe_extensions = []
+            safe_extensions = []
             for extension in extensions:
-                if extension[0] == ".":
-                    periodsafe_extensions.append(extension[1:])
-                else:
-                    periodsafe_extensions.append(extension)
-                extensions_regex = "|".join(periodsafe_extensions)
-                regex = f".*?/{top_level_filename}\\.({extensions_regex})$"
+                extension = _extension_safety(extension)
+                safe_extensions.append(extension)
+            extensions_regex = "|".join(safe_extensions)
```

Comment:
eh,can't do suggestion since covers a deleted line but could be just following for a more concise etc code

```python
extensions_regex = "|".join(map(_extension_safety, extensions))
```
right?

Reply:
yep!